pkg/core/interops.go

```diff
@@ -43,6 +43,7 @@ var systemInterops = []interop.Function{
 	{Name: interopnames.SystemContractCall, Func: contract.Call, Price: 1 << 15,
 		RequiredFlags: callflag.AllowCall, ParamCount: 4},
 	{Name: interopnames.SystemContractCallNative, Func: native.Call, Price: 0, ParamCount: 1},
+	{Name: interopnames.SystemContractCreateMultisigAccount, Func: contractCreateMultisigAccount, Price: 1 << 8, ParamCount: 1},
```

Comment:
I think it is should be a bit more expensive that `Standard` equivalent. `1 << 9`?

Reply:
In C# node the price is `1<<8`, but I also thought about it. Create an issue?

Reply:
Prices are not that important for us though key decoding is not free and it there can be a lot of keys there, so fair price relative to Standard is probably higher.